src/data/fields.php

```diff
@@ -180,7 +180,7 @@ private function prepare_html( $html ) {
 	 */
 	private function parse_post_content( $post ) {
 		// Prepare the HTML for parsing.
-		$html = $this->prepare_html( $post->post_content );
+		$html = $this->prepare_html( $post->contentRaw );
```

Comment:
thanks for the PR @cojennin! can you explain this change a bit?

Reply:
Hey, yea, I've been using this for a bit now and it could do with some more thought.

With WPGraphQL v0.3.0, [resolve](https://github.com/Quartz/wp-graphql-content-blocks/pull/2/commits/42a2677dd22118659d73151e59fa8b6aa5728066#diff-8596774a19d60da860533c7e656dbdf1R339) is now getting a `\WPGraphQL\Model\Post` instead of a `WP_Post`. [\WPGraphQL\Model\Post](https://github.com/wp-graphql/wp-graphql/blob/v0.3.0/src/Model/Post.php#L269-L279) doesn't expose `post_content`, it only exposes `contentRaw` and `contentRendered`.

I switched to using `contentRaw` here, but I think another option would be to get do a `get_post` to get the `WP_Post` object instead of dealing with `\WPGraphQL\Model\Post`. Avoids having to tangle with the capability checks included in `\WPGraphQL\Model\Post`. 

Or maybe [resolve](https://github.com/Quartz/wp-graphql-content-blocks/pull/2/commits/42a2677dd22118659d73151e59fa8b6aa5728066#diff-8596774a19d60da860533c7e656dbdf1R339) can be called in a way that it continues to receive `WP_Post`?